UndertaleModLib/Models/UndertaleSprite.cs

```diff
@@ -16,25 +16,41 @@ public enum AnimSpeedType : uint
 [PropertyChanged.AddINotifyPropertyChangedInterface]
 public class UndertaleSpineTextureEntry : UndertaleObject, IDisposable
 {
+    /// <summary>
+    /// The width of the Spine atlas in pixels.
+    /// </summary>
     public int PageWidth { get; set; }
+    
+    /// <summary>
+    /// The height of the Spine atlas in pixels.
+    /// </summary>
     public int PageHeight { get; set; }
-    public byte[] PNGBlob { get; set; }
+    
+    /// <summary>
+    /// Can be raw GM QOI bytes or PNG bytes.
+    /// </summary>
+    public byte[] TexBlob { get; set; }
+    
+    /// <summary>
+    /// Returns true if TexBlob contains a GMQOI texture.
+    /// </summary>
+    public bool IsQOI => TexBlob != null && TexBlob.Length > 7 && TexBlob[0] == 102 && TexBlob[1] == 105 && TexBlob[2] == 111 && TexBlob[3] == 113;
```

Comment:
Might wanna add as a comment that it checks for the QOI header here? Makes it a bit more understandable why it checks for "random numbers" here